netshare/generate/model_generate.py

```diff
@@ -112,44 +119,28 @@ def merge_attr(config_group: dict, configs: List[dict]) -> None:
             os.path.join(attr_raw_npz_folder, "chunk_id-{}.npz".format(chunkid))
         )["data_attribute"]
 
-        if num_chunks > 1 and word2vec_size:
+        if num_chunks > 1:
             for row in raw_attr_chunk:
-                # if row[bit_idx_flagstart] < row[bit_idx_flagstart+1]:
                 if (
-                    row[bit_idx_flagstart] < row[bit_idx_flagstart + 1]
-                    and row[bit_idx_flagstart + 2 * chunkid + 2]
-                    < row[bit_idx_flagstart + 2 * chunkid + 3]
+                    row[col_idx_of_session_start] < row[col_idx_of_session_start + 1]
+                    and row[col_idx_of_session_start + 2 * chunkid + 2]
+                    < row[col_idx_of_session_start + 2 * chunkid + 3]
                 ):
                     # this chunk
-                    row_this_chunk = list(copy.deepcopy(row)[:bit_idx_flagstart])
-                    row_this_chunk += [0.0, 1.0]
-                    row_this_chunk += [1.0, 0.0] * (chunkid + 1)
-                    for i in range(chunkid + 1, num_chunks):
-                        if (
-                            row[bit_idx_flagstart + 2 * i + 2]
-                            < row[bit_idx_flagstart + 2 * i + 3]
-                        ):
-                            row_this_chunk += [0.0, 1.0]
-                        else:
-                            row_this_chunk += [1.0, 0.0]
-                    # dict_chunkid_attr[chunkid].append(row_this_chunk)
```

Comment:
the `row_this_chunk` is constructed but not used in the following logic. I just deleted them.